packages/core-data/src/entity-types/index.ts

```diff
@@ -11,6 +11,8 @@ import type { NavMenuItem } from './nav-menu-item';
 import type { Page } from './page';
 import type { Plugin } from './plugin';
 import type { Post } from './post';
+import type { PostStatusObject } from './post-status';
+import type { Base } from './base';
```

Comment:
Nit: might make sense to keep the alphabetic order here.

Reply:
Done. Thanks